### ISSUE TITLE
Confirm that we do have tests for no-batch-dim support and the batching rules actually support them

### ISSUE THREAD

**Issue by zou3519** [pytorch/functorch#443]
Context: https://github.com/pytorch/pytorch/issues/60585 adds no-batch-dim support for a lot of nn.functional operations. However, many of them didn't add OpInfos to test this functionality.

Because they didn't add OpInfos, we do not actually know if our vmap batching rules support no-batch-dim or not because our tests are auto-generated from OpInfos.

This task is to make sure the batching rules support no-batch-dim for those operators.

**Comment by zou3519** [comment]
Was confirmed
